test/files/neg/t6810.scala

```diff
@@ -15,14 +15,14 @@ trait t6810 {
   val B = s"""
 """                   // or the same for interpolated strings
 
-  import scala.compat.Platform.EOL
-  val `\u000A` = EOL  // backquoted identifiers are arbitrary string literals
+  val `\u000A` = System.lineSeparator()  // backquoted identifiers are arbitrary string literals
   val `
-` = EOL               // not raw string literals aka triple-quoted, multiline strings
+` = System.lineSeparator()               // not raw string literals aka triple-quoted, multiline strings
 
   val firebreak = 42  // help parser recovery, could also use rbrace
 
   val a = '\u000D'    // similar treatment of CR
-  val b = '
'        // CR seen as EOL by scanner
+  val b = '
```

Comment:
hm...

Reply:
Editor rewrote it?

I see it's a line separator but not a NL token: `val i = 42^Mval j = 17` is two lines but there is no inferred NL.

Reply:
Was this not restored? It tests CR.

Reply:
@som-snytt Sorry, I don't know how to type it. IDEA can't properly render it.

Reply:
I'd suggest just skipping it. If the warning appears in the test output, you can update the check file with `partest --update-check`.

Reply:
I could take a try on my windows laptop.